src/main/java/study/lotto/dto/DivisionDetails.java

```diff
@@ -36,6 +34,8 @@ public int getMatchCount() {
         return matchCount;
     }
 
+    public boolean isBonusMatch() { return bonusMatch; }
```

Comment:
```suggestion
    public boolean isBonusMatch() { 
        return bonusMatch; 
    }
```

한 라인으로 작성해주신 이유가 있을까요?

Reply:
정말 꼼꼼한 리뷰 감사드립니다! 👍 
인텔리제이에서 자동 포메팅을 사용하는데 이 부분을 추가하면서 단축키를 누르지 않은것 같네요. 인텔리제이에서 짧은 함수는 자동으로 접혀서 한줄로 보여줘서 이것 또한 그런지 알고 넘어간것 같아요.